TrackletAlgorithm/DummyMessageLogger.h

```diff
@@ -0,0 +1,40 @@
+#ifndef MessageLogger_MessageLogger_h
```

Comment:
What is the point of the line "#ifndef MessageLogger_MessageLogger_h", given that this pragma is never defined anywhere?

Reply:
It was an attempt to use both the MessageLogger, which defines this macro, and my dummy but it's not needed now so I'll remove it.